repos/public-user/src/Settings/PublicUserSettings.php

```diff
@@ -29,6 +43,32 @@ public function __construct(
         $this->currentSiteSettings = $currentSiteSettings;
     }
 
+    const APPLY_CUSTOM_SITE_PERMISSION_TO_ROLES = [
+        'viewer',
+        'limited-transcriber',
+        'limited-reviewer',
+        'transcriber',
```

Comment:
In Omeka there are 3 levels of functionality, `admin`, `editor` and `viewer`. They are assigned numbers 3-1. Functionality and permissions are then checked using a simple greater/less than. Effectively `editor` gets all the viewer permissions and the editor ones, and admin gets them all. This is an extension of that, these are the permissions that need to be checked on top of this structure. Everything that's not an `admin` or `editor` has base level permissions of a `viewer`. This is one dimension of the permission model in the configuration, with the `CUSTOM_SITE_PERMISSIONS` being the other dimension.